Xamarin.Forms.Platform.Android/Extensions/ImageViewExtensions.cs

```diff
@@ -65,8 +67,10 @@ internal static class ImageViewExtensions
 							using (var drawable = await imageView.Context.GetFormsDrawableAsync(newImageSource))
 							{
 								// only set the image if we are still on the same one
-								if (!imageView.IsDisposed() && SourceIsNotChanged(newView, newImageSource))
+								if (!imageView.IsDisposed() && SourceIsNotChanged(newView, newImageSource) && drawable != null)
 									imageView.SetImageDrawable(drawable);
+								else
```

Comment:
should this be

```C#
else if (!imageView.IsDisposed() && SourceIsNotChanged(newView, newImageSource) && drawable == null)
									await imageView.SetImagePlaceholder(newView.ErrorSource);
```

- If the imageView is disposed we don't want to set anything
- If the source has changed then we don't want to do anything and there's an update pending
- we only want to update if those two conditions are still true and drawable == null right?

so maybe this?
```C#
if (!imageView.IsDisposed() && SourceIsNotChanged(newView, newImageSource))
								{
									if(drawable == null)
										await imageView.SetImagePlaceholder(newView.ErrorSource);
									else
										imageView.SetImageDrawable(drawable);
								}
```